run/README.namelist

```diff
@@ -1278,37 +1285,45 @@ Options for use with the Noah-MP Land Surface Model:
+                                                     1 = Liu, et al. 2016
+                                                     2 = Gecros (Genotype-by-Environment interaction on CROp growth Simulator) Yin and van Laar, 2005
  
  opt_irr                            = 0,        !  options for irrigation scheme
                                                            geogrid must have been run with GEOGRID.TBL.ARW.noahmp, use with caution
-	                                             0 -> No irrigation
-                                                     1 -> Irrigation ON
-                                                     2 -> irrigation trigger based on crop season Planting and harvesting dates
-                                                     3 -> irrigation trigger based on LAI threshold
+	                                             0 = No irrigation
+                                                     1 = Irrigation ON
+                                                     2 = irrigation trigger based on crop season Planting and harvesting dates
+                                                     3 = irrigation trigger based on LAI threshold
  opt_irrm                           = 0,        !  options for irrigation method (only if opt_irr > 0)
                                                            geogrid must have been run with GEOGRID.TBL.ARW.noahmp, use with caution
-                                                     0 -> method based on geo_em fractions (all three methods are ON)
-                                                     1 -> sprinkler method
-                                                     2 -> micro/drip irrigation
-                                                     3 -> surface flooding  
+                                                     0 = method based on geo_em fractions (all three methods are ON)
+                                                     1 = sprinkler method
+                                                     2 = micro/drip irrigation
+                                                     3 = surface flooding  
+ opt_tdrn                           = 0,        ! Noah-MP tile drainage option (currently only tested and works with opt_run=3)
+                                                     geogrid must have been run with GEOGRID.TBL.ARW.noahmp, use with caution
+                                                     0 = No tile drainage
+                                                     1 = Simple drainage
+                                                     2 = Hooghoudt's equation based tile drainage
+ soiltstep                          = 0.0,      ! Noah-MP soil process timestep (seconds) for solving soil water and temperature
+                                                     0 = default, the same as main NoahMP model timestep
+                                                     N * dt_noahmp, typically 15min or 30min
```

Comment:
@cenlinhe Can you add noahmp_output, noahmp_acc_dt, and possibly opt_infdv, opt_tdrn to this file?